lua/neogit/config.lua

```diff
@@ -10,6 +10,10 @@ M.values = {
     item = { ">", "v" },
     section = { ">", "v" }
   },
+  notify_opts = {
```

Comment:
I am not really a fan of having this in the configuration.

Reply:
I thought users would like to override icon, for example. But I can hardcode it, yes.